bindings.go

```diff
@@ -454,6 +478,26 @@ func (d Device) AverageGPUUtilization(since time.Duration) (uint, error) {
 	return uint(n), errorString(r)
 }
 
+func (d Device) EncoderUtilization() (uint, uint, error) {
+	if C.nvmlHandle == nil {
+		return 0, 0, errLibraryNotLoaded
+  }
+  var n C.uint
+  var sp C.uint
+  r := C.nvmlDeviceGetEncoderUtilization(d.dev, &n, &sp)
+  return uint(n), uint(sp), errorString(r) 
+}
+
```

Comment:
Can you add doc comments? What are the two values getting returned.

Reply:
OK, I will add doc comments :)